lib/actions/admin/updateUser.js

```diff
@@ -11,12 +11,11 @@ module.exports = function (req, res) {
       user.isMember = !!req.body.isMember
       user.isCurator = !!req.body.isCurator
       user.isAdmin = !!req.body.isAdmin
-      return user.save()
+      user.save()
```

Comment:
Sorry, I missed something. It should be `user.save.then(() => { res.... })` for the 200, so that it will only send a 200 if the save succeeds, otherwise it will catch the error and send the 500. The way it is now, it will send the 200 without waiting for the `destroy` or `save` to complete, and if they have an error it will throw the error about being unable to set headers.